src/phylum/ci/ci_base.py

```diff
@@ -376,7 +387,7 @@ def _check_prerequisites(self) -> None:
             raise SystemExit(msg)
 
     @progress_spinner("Ensuring a Phylum project exists")
-    def ensure_project_exists(self) -> None:
+    def _ensure_project_exists(self) -> None:
```

Comment:
Curious what the `_` prefix is for.

Reply:
Single leading underscores in variable and function names are a convention that indicates "internal use only." A much better and lengthier discussion about underscore usage in Python can be found [in this blog post](https://dbader.org/blog/meaning-of-underscores-in-python).